api/anonymous.js

```diff
@@ -41,6 +41,19 @@ routes.post('/user/register', async (req, res, next) => {
     }
 });
 
+/*
+routes.post('/user/profile', async (account_id, val) => {
+
+	try {
+		account_mgmt.meetingSet(account_id, val);
+		res.status(200).send('Success');
+	} catch (error){
```

Comment:
why commented?

Reply:
this s outdated and needs to be completely removed. ill do that now